[2024-12-06T11:31:44.063+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T11:31:44.072+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: telemadrid_schedule_scraper_v2.PushAllInOne scheduled__2024-02-12T11:32:00+00:00 [queued]>
[2024-12-06T11:31:44.077+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: telemadrid_schedule_scraper_v2.PushAllInOne scheduled__2024-02-12T11:32:00+00:00 [queued]>
[2024-12-06T11:31:44.078+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 5
[2024-12-06T11:31:44.086+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): PushAllInOne> on 2024-02-12 11:32:00+00:00
[2024-12-06T11:31:44.089+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=452) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T11:31:44.090+0000] {standard_task_runner.py:72} INFO - Started process 456 to run task
[2024-12-06T11:31:44.091+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'telemadrid_schedule_scraper_v2', 'PushAllInOne', 'scheduled__2024-02-12T11:32:00+00:00', '--job-id', '772', '--raw', '--subdir', 'DAGS_FOLDER/dag_telemadrid_v2.py', '--cfg-path', '/tmp/tmpakhlahdx']
[2024-12-06T11:31:44.092+0000] {standard_task_runner.py:105} INFO - Job 772: Subtask PushAllInOne
[2024-12-06T11:31:44.121+0000] {task_command.py:467} INFO - Running <TaskInstance: telemadrid_schedule_scraper_v2.PushAllInOne scheduled__2024-02-12T11:32:00+00:00 [running]> on host d88abb179e8d
[2024-12-06T11:31:44.167+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='telemadrid_schedule_scraper_v2' AIRFLOW_CTX_TASK_ID='PushAllInOne' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T11:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T11:32:00+00:00'
[2024-12-06T11:31:44.168+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T11:31:46.453+0000] {logging_mixin.py:190} INFO - Archivo actualizado correctamente.
[2024-12-06T11:31:46.454+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T11:31:46.461+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T11:31:46.461+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=telemadrid_schedule_scraper_v2, task_id=PushAllInOne, run_id=scheduled__2024-02-12T11:32:00+00:00, execution_date=20240212T113200, start_date=20241206T113144, end_date=20241206T113146
[2024-12-06T11:31:46.509+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T11:31:46.517+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
